charts/redpanda/helpers_test.go

```diff
@@ -0,0 +1,153 @@
+package redpanda_test
+
+import (
+	"testing"
+
+	"github.com/redpanda-data/helm-charts/charts/redpanda"
+	"github.com/stretchr/testify/assert"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/utils/ptr"
+)
+
+func TestStategicMergePatch(t *testing.T) {
```

Comment:
Could you add EnvVar example of appending overwrite on top of what original has, just to visualise this in one of those test cases?

Reply:
Done! And it caught a but 😅